tools/cli/internal/changelog/sunset_test.go

```diff
@@ -126,7 +126,7 @@ func TestNewOasDiffEntriesFromSunsetEndpoints(t *testing.T) {
 			Operation:   "GET",
 			OperationID: "listStreamInstances",
 			Path:        "/api/atlas/v2/groups/{id}/streams",
-			Text:        "endpoint removed",
+			Text:        "endpoint with API Version '2023-02-01' was removed as it has reached its sunset date '2023-07-12'",
```

Comment:
I am not sure what is the suggested message 🤔 
do you mean from `endpoint with API Version '2023-02-01' was removed as it has reached its sunset date '2023-07-12'` to endpoint with API Version '2023-02-01' (sunset date '2023-07-12)'

Reply:
no, only `endpoint removed (sunset date '2023-07-12)'`

Reply:
I think the API Version is important otherwise it may seem that the endpoint as a whole was removed but in reality only one version was removed. Am I missing something?